assets/app/features/upgrade/templates/upgrade-administration-server-page.jade

```diff
@@ -10,4 +10,4 @@ button.btn.btn-success.center-block(
 )
     suse-lazy-spinner(delay="2000", active="upgradeAdminUpgradeVm.adminUpgrade.running",
         visible="upgradeAdminUpgradeVm.adminUpgrade.spinnerVisible")
-    span(translate='') upgrade.steps.upgrade-admin.form.upgrade-admin
+    span(translate='', ng-bind="upgradeAdminUpgradeVm.finish_text") upgrade.steps.upgrade-admin.form.upgrade-admin
```

Comment:
This was only done as a test for the text change on upgrade finish and for the translation question. It will be removed in the next comimt :)